R/data_maggie.R

```diff
@@ -0,0 +1,464 @@
+# TODO: some variables are not defined in articles or datasets-- is there another source I can check?
+#   For "Complete history of the NBA" I wasn't sure about the meaning of the variable "neutral" (consulted outside sources)
+#   For "What the world think of trump" I couldn't figure out dk_refused-- maybe percent of respondents who refused to answer?
+#   For the "How popular is Donald Trump" first dataset I couldn't figure out "multiversions" variable (maybe if multiple versions of the poll were sent out?) and second dataset I couldn't figure out what the models were based on (it seemed like it should be one model per poll but there were far fewer models than polls)
```

Comment:
If you can't deduce it, leave it in the help file as "undefined"

Reply:
Also, don't forget to build the package early and build it often to get a preview of how everything looks! Got to test errything! For example, be sure to sanity check all the help files `?nba_carmelo` to see that things look good.

You can click the "Build and Reload" button here to build the package. Don't forget to turn on roxygen help file building as noted [here](http://r-pkgs.had.co.nz/man.html#man-workflow-2) first

<img width="625" alt="build" src="https://user-images.githubusercontent.com/4574203/35124682-0015364a-fc75-11e7-942d-514fec6ba84b.png">